bookwyrm/templates/user/user.html

```diff
@@ -4,6 +4,10 @@
 
 {% block title %}{{ user.display_name }}{% endblock %}
 
+{% block head_links %}
+    <link rel="alternate" type="application/rss+xml" href="{{ user.local_path }}/rss" title="{{ user.display_name }}" />
```

Comment:
The bare user display name could be a little terse for subscribers. Consider including the instance name. To make this internationalised, you can do something like:

```python
{% translate "on" as on %}
...
{% block head_links %}
    <link rel="alternate" type="application/rss+xml" href="{{ user.local_path }}/rss" title="{{ user.display_name }} {{ on }} {{ site.name }}" />


```